mkdocs.yml

```
@@ -1,9 +1,9 @@
-site_name: stac-fastapi
-site_description: STAC FastAPI.
+site_name: stac-fastapi-sqlachemy
+site_description: SQLAlchemy backend for stac-fastapi.
 
 # Repository
-repo_name: 'stac-utils/stac-fastapi'
-repo_url: 'https://github.com/stac-utils/stac-fastapi'
+repo_name: 'stac-utils/stac-fastapi-sqlalchemy'
+repo_url: 'https://github.com/stac-utils/stac-fastapi-sqlalchemy'
 edit_uri: 'blob/master/docs/src/'
```

```suggestion
edit_uri: 'blob/main/docs/src/'
```

I realized I need to change this in `stac-fastapi-pgstac`, too...